README.md

```diff
@@ -267,17 +267,17 @@ This plugin creates device plugin endpoints based on the configurations given in
 | "resourcePrefix"  | N        | Endpoint resource prefix name override. Should not contain special characters                                                          | string Default : "intel.com"                          | "yourcompany.com"                                                      |
 | "deviceType"      | N        | Device Type for a resource pool.                                                                                                       | string value of supported types. Default: "netDevice" | Currently supported values: "accelerator", "netDevice", "auxNetDevice" |
 | "excludeTopology" | N        | Exclude advertising of device's NUMA topology                                                                                          | bool Default: "false"                                 | "excludeTopology": true                                                |
-| "selectors"       | N        | A map of device selectors. The "deviceType" value determines the "selectors" options.                                                  | json object as string Default: null                   | Example: "selectors": {"vendors": ["8086"],"devices": ["154c"]}        |
+| "selectors"       | N        | An array of device selector maps. The "deviceType" value determines the device selector options.                                                  | json object as string Default: null                   | Example: "selectors": [{"vendors": ["8086"],"devices": ["154c"]}]        |
```

Comment:
Also i think we should mention we support two forms here, 
a selector map or  a list of selector map WDYT ?

Reply:
Agreed.

Reply:
Updated.